### PR TITLE
fix: fix examples to always set service url

### PR DESCRIPTION
**Pull request by mamoonraja**
This PR does the following:

- Makes sure we set URL in examples.
- Fix semantic release.

### CODE REVIEW

File: Examples/ExampleAssistantV1.cs [hunk 1]
```diff
@@ -114,6 +114,7 @@ private IEnumerator CreateService()
                 yield return null;
 
             service = new AssistantService("2019-02-18", authenticator);
+            service.SetServiceUrl("{serviceUrl}");
```
Comment (mediumTaj) [inline Examples/ExampleAssistantV1.cs]: Do we need to add the null check here 

```cs
if (!string.IsNullOrEmpty(serviceUrl))
{
    service.SetServiceUrl(serviceUrl);
}
```

File: Examples/ExamplePersonalityInsightsV3.cs [hunk 2]
```diff
@@ -55,6 +55,7 @@ private IEnumerator CreateService()
                 yield return null;
 
             service = new PersonalityInsightsService("2019-02-18", authenticator);
+            service.SetServiceUrl("{serviceUrl}");
```
Comment (mediumTaj) [inline Examples/ExamplePersonalityInsightsV3.cs]: Do we need to add the null check here 

```cs
if (!string.IsNullOrEmpty(serviceUrl))
{
    service.SetServiceUrl(serviceUrl);
}
```

File: Scripts/Services/SpeechToText/V1/SpeechToTextService.cs [hunk 3]
```diff
@@ -52,13 +52,11 @@ public bool DisableSslVerification
         /// <summary>
         /// SpeechToTextService constructor.
         /// </summary>
-        
         public SpeechToTextService() : this(ConfigBasedAuthenticatorFactory.GetAuthenticator(serviceId)) {}
 
         /// <summary>
         /// SpeechToTextService constructor.
         /// </summary>
-
```
Comment (mediumTaj) [inline Scripts/Services/SpeechToText/V1/SpeechToTextService.cs]: Was this regenerated?

Comment (mamoonraja) [reply Scripts/Services/SpeechToText/V1/SpeechToTextService.cs]: yeah, probably should not change it in this pr

**Pull request merged**